front/app/components/FormBuilder/components/FormBuilderToolbox/ToolboxItem.tsx

```diff
@@ -52,40 +57,61 @@ const ToolboxItem = ({
   disabled,
   ...rest
 }: Props) => {
+  const { formatMessage } = useIntl();
+
   if (fieldsToExclude && inputType && fieldsToExclude.includes(inputType)) {
     return null;
   }
 
   return (
-    <StyledBox
-      display="flex"
-      px="18px"
-      py="18px"
-      onClick={onClick}
-      width="100%"
-      m="0px"
-      alignItems="center"
-      // remove the role attribute when we add drag and drop functionality
-      role="button"
-      data-cy={rest['data-cy']}
-      disabled={!!disabled}
+    <Tippy
+      interactive={true}
+      placement={'left'}
+      disabled={!disabled}
+      theme={'dark'}
+      content={
+        <Text my="4px" color="white">
+          {formatMessage(messages.disabledFieldTooltip)}
```

Comment:
Maybe this message should come from the config instead? I can imagine we'd have a use for tooltips on disabled fields for other participation methods as well :thinking:

Reply:
🤔 We could add it when the time comes